app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java

```diff
@@ -57,8 +57,7 @@
 
 
     /**
-     * Public constructor.
-     *
+     * Public constructor. *
```

Comment:
nit: the star (`*`) goes on the next line ;)

Reply:
Fixed